azure-mgmt-servicebus/tests/test_azure_mgmt_servicebus_topic.py

```diff
@@ -32,8 +32,8 @@ def test_sb_topic_curd(self, resource_group, location):
         # Create a Namespace
         namespace_name = "testingpythontestcasetopic"
 
-        namespaceparameter = SBNamespace(location, {'tag1': 'value1', 'tag2': 'value2'}, SBSku(SkuName.standard))
-        creatednamespace = self.servicebus_client.namespaces.create_or_update(resource_group_name, namespace_name, namespaceparameter,None,True).output
+        namespaceparameter = SBNamespace(location=location,tags={'tag1': 'value1', 'tag2': 'value2'},sku=SBSku(name=SkuName.standard))
+        creatednamespace = self.servicebus_client.namespaces.create_or_update(resource_group_name, namespace_name, namespaceparameter)._polling_method._operation.resource
```

Comment:
Don't do that, that's really bad. And it's why the CI is breaking.
If you wish not to wait, use `polling=False`. If you wish to wait, call `result()`